uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Form/AbsenceType.tpl

```diff
@@ -183,7 +183,7 @@
 
                 function hideToilExpiration() {
                   document.getElementById('accrual_expiration_duration').value = '';
-                  $('#accrual_expiration_unit').select2('val', '');
```

Comment:
Does this mean the problem would also happen if someone disabled TOIL for an Absence Type. If so, please mention that in the ticket description and tell QA to also check that while testing

Reply:
True, added to the ticket.